amida-tech/hl7-telescoper-client
# Feature/message viewer

# what does this do
oh man so much - sorry this is way too big for a normal PR
- lists files
- upload files
- display messages
- fetch messages
- highlight content of messages

# how to test
make sure you have the getMessage endpoint merged for the backend first
go through entire user flow
- signup
- login
- upload file
- upload anotha one
- click "explore" next to a file
- see first message in file
- hover over things
- click on things in list and in raw, make sure correct things highlight
- click the up arrow next to message num at top right, make sure the next message is fetched and displayed (untested as of Friday 4pm)
